### DISCUSSION TITLE
Handlebars 란 무엇인가

### DISCUSSION THREAD

**Discussion by starshb** [starshb/pharmacy_Find#11] [Q&A] [answered]
Handlebars

**Comment by tjdud22** [comment, marked as answer]
Mustache를 기반으로 구현한 간단한 templating language라고 한다

 Handlebars는 Semantic templates을 구축하는데 필요한 기능을 제공한다.
Mustache-compatible(호환)하여 Mustache를 핸들바로 교체하고 현재 템플릿을 계속 사용 사용할 수 있다.
JavaScript함수로 컴파일하기 때문에 다른 템플릿엔진보다 템플릿 실행을더 빠르게 할 수 있다.
